driver/image_normalize_driver.hpp

```diff
@@ -0,0 +1,347 @@
+        miopenCreateTensorDescriptor(&meanTensorDesc);
+        miopenCreateTensorDescriptor(&stdvarTensorDesc);
+
+        data_type = miopen_type<Tgpu>{};
+    }
+
+    int AddCmdLineArgs() override;
+    int ParseCmdLineArgs(int argc, char* argv[]) override;
+    InputFlags& GetInputFlags() override { return inflags; }
+
+    int GetandSetData() override;
+    int AllocateBuffersAndCopy() override;
+
+    int RunForwardGPU() override;
+    int RunForwardCPU();
+
+    int RunBackwardGPU() override;
+    int RunBackwardCPU();
+
+    int VerifyForward() override;
+    int VerifyBackward() override;
+
+    ~ImageNormalizeDriver() override
+    {
+        miopenDestroyTensorDescriptor(inputTensorDesc);
+        miopenDestroyTensorDescriptor(outputTensorDesc);
+        miopenDestroyTensorDescriptor(meanTensorDesc);
+        miopenDestroyTensorDescriptor(stdvarTensorDesc);
+    }
+
+private:
+    InputFlags inflags;
+    int forw;
+
+    miopenTensorDescriptor_t inputTensorDesc;
+    miopenTensorDescriptor_t outputTensorDesc;
+    miopenTensorDescriptor_t meanTensorDesc;
+    miopenTensorDescriptor_t stdvarTensorDesc;
+
+    std::unique_ptr<GPUMem> input_gpu;
+    std::unique_ptr<GPUMem> output_gpu;
+    std::unique_ptr<GPUMem> mean_gpu;
+    std::unique_ptr<GPUMem> stdvar_gpu;
+
+    std::vector<Tgpu> input_host;
+    std::vector<Tgpu> output_host;
+    std::vector<Tgpu> mean_host;
+    std::vector<Tgpu> stdvar_host;
+
+    std::vector<Tref> out_ref;
+};
+
+template <typename Tgpu, typename Tref>
+int ImageNormalizeDriver<Tgpu, Tref>::AddCmdLineArgs()
+{
+    inflags.AddInputFlag("forw", 'F', "1", "Run only the forward pass", "int");
+    inflags.AddTensorFlag("input", 'I', "1x3x96x96", "Input Tensor Size");
+    inflags.AddTensorFlag("mean", 'M', "3", "Mean Tensor Sequence Size");
+    inflags.AddTensorFlag("stdvar", 'S', "3", "Stdvar Tensor Sequence Size");
+    inflags.AddInputFlag("contiguous", 'Z', "1", "Use Contiguous Tensors", "int");
+
+    inflags.AddInputFlag("iter", 'i', "10", "Number of iterations", "int");
+    inflags.AddInputFlag("verify", 'V', "1", "Verify Each Layer (Default=1)", "int");
+    inflags.AddInputFlag("time", 't', "0", "Time Each Layer (Default=0)", "int");
+    inflags.AddInputFlag(
+        "wall", 'w', "0", "Wall-clock Time Each Layer, Requires time == 1 (Default=0)", "int");
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageNormalizeDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
+{
+    inflags.Parse(argc, argv);
+
+    if(inflags.GetValueInt("time") == 1)
+    {
+        miopenEnableProfiling(GetHandle(), true);
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageNormalizeDriver<Tgpu, Tref>::GetandSetData()
+{
+    TensorParameters input_vec = inflags.GetValueTensor("input");
+
+    assert(input_vec.lengths.size() == 4 || input_vec.lengths.size() == 3);
+    if(input_vec.lengths.size() == 3)
+    {
+        // n=1
```

Comment:
Please remove comment

Reply:
Changed the comment to better reflect why an insertion of `n=1` is necessary